code/modules/clothing/outfits/standard.dm

```diff
@@ -121,21 +121,57 @@
 
 	uniform = /obj/item/clothing/under/costume/pirate
 	shoes = /obj/item/clothing/shoes/sneakers/brown
-	suit = /obj/item/clothing/suit/pirate
-	head = /obj/item/clothing/head/bandana
+	suit = /obj/item/clothing/suit/pirate/armored
+	head = /obj/item/clothing/head/bandana/armored
+	ears = /obj/item/radio/headset/syndicate
 	glasses = /obj/item/clothing/glasses/eyepatch
 
+/datum/outfit/pirate/captain
+	name = "Space Pirate Captain"
+
+	head = /obj/item/clothing/head/pirate/armored
+
 /datum/outfit/pirate/space
+	name = "Space Pirate (EVA)"
 	suit = /obj/item/clothing/suit/space/pirate
 	head = /obj/item/clothing/head/helmet/space/pirate/bandana
 	mask = /obj/item/clothing/mask/breath
 	suit_store = /obj/item/tank/internals/oxygen
-	ears = /obj/item/radio/headset/syndicate
 	id = /obj/item/card/id
 
 /datum/outfit/pirate/space/captain
+	name = "Space Pirate Captain (EVA)"
+
 	head = /obj/item/clothing/head/helmet/space/pirate
 
+/datum/outfit/pirate/silverscale
+	name = "Silver Scale Member"
+
+	head = /obj/item/clothing/head/collectable/tophat
+	glasses = /obj/item/clothing/glasses/monocle
+	uniform = /obj/item/clothing/under/suit/charcoal
+	shoes = /obj/item/clothing/shoes/laceup
+	suit = /obj/item/clothing/suit/armor/vest/alt
+	gloves = /obj/item/clothing/gloves/color/black
+
+	id = /obj/item/card/id/silver
+
+	var/scale_assignment = "Silver Scale Member"
+
+/datum/outfit/pirate/silverscale/post_equip(mob/living/carbon/human/H)
+	..()
+	var/obj/item/card/id/W = H.wear_id
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names